projects/igniteui-angular/src/lib/navbar/navbar.component.html

```diff
@@ -1,8 +1,12 @@
 <nav class="igx-navbar" role="navigation" [attr.aria-labelledby]="titleId">
     <div class="igx-navbar__left">
-        <igx-icon (click)="_triggerAction()" fontSet="material" *ngIf="isActionButtonVisible">{{actionButtonIcon}}</igx-icon>
-        <ng-content select="igx-action-icon"></ng-content>
-        <h1 class="igx-navbar__title" [attr.id]="titleId">{{ title }}</h1>
+        <igx-icon (click)="_triggerAction()" fontSet="material" *ngIf="isActionButtonVisible">{{actionButtonIcon}}
+        </igx-icon>
+        <ng-content select="[igxNavbarAction]"></ng-content>
+        <h1 *ngIf="!isTitleContentVisible" class="igx-navbar__title" [attr.id]="titleId">{{ title }}</h1>
+    </div>
+    <div class="igx-navbar__middle">
+        <ng-content select="[igxNavbarTitle]"></ng-content>
```

Comment:
We don't want a middle area. Move the igxNavbarTitle directive next to the `h1` title.

Reply:
Done